### PR TITLE
Fix compiler warnings

### PR DESCRIPTION
**Pull request by herostrat**
SSIA

### CODE REVIEW

File: src/common/VirtualTerminal.cpp [hunk 1]
```diff
@@ -210,7 +210,9 @@ namespace SDDM {
 
                 // Clear VT
                 static const char *clearEscapeSequence = "\33[H\33[2J";
-                write(vtFd, clearEscapeSequence, sizeof(clearEscapeSequence));
+                if (write(vtFd, clearEscapeSequence, sizeof(clearEscapeSequence)) == -1) {
```
Comment (Vogtinator) [inline src/common/VirtualTerminal.cpp:213]: Technically checking for success requires `== sizeof(clearEscapeSequence)`, but then errno isn't set and that's even more annoying. So IMO this is fine

**Review by Vogtinator — APPROVED** [review]

File: src/common/VirtualTerminal.cpp (hunk 1 above)
Comment (herostrat) [reply src/common/VirtualTerminal.cpp:213]: You're right, I wasn't sure if it is worth the effort.
I can change it if you prefer :)

Comment (Vogtinator) [reply src/common/VirtualTerminal.cpp:213]: No need!

**Review by aleixpol — APPROVED** [review]

**Comment by herostrat** [comment]
Thanks!

**Pull request merged**